lua/nvim-tree/actions/file-popup.lua

```diff
@@ -0,0 +1,64 @@
+local a = vim.api
+local uv = vim.loop
+
+local M = {}
+
+local function get_formatted_lines(cwd)
+  local stats = uv.fs_stat(cwd)
+  local fpath = ' fullpath: ' .. cwd
+  local created_at = ' created:  ' .. os.date("%x %X", stats.birthtime.sec)
+  local modified_at = ' modified: ' .. os.date("%x %X", stats.mtime.sec)
+  local accessed_at = ' accessed: ' .. os.date("%x %X", stats.atime.sec)
+  local size = ' size:     ' .. stats.size .. ' bytes'
+
+  return {
+    fpath,
+    size,
+    accessed_at,
+    modified_at,
+    created_at,
+  }
+end
+
+local winnr = nil
+
+local function setup_window(lines)
+  local max_width = vim.fn.max(vim.tbl_map(function(n) return #n end, lines))
+  winnr = a.nvim_open_win(0, false, {
+    col = 1,
+    row = 1,
+    relative = "cursor",
+    width = max_width + 1,
+    height = #lines,
+    border = 'shadow',
```

Comment:
This (border) feels bit too opinionated, could it be made configurable maybe?

Reply:
this is just a first draft, this feature can be extended as needed. Adding a config for `actions.file_popup = { ... }` would be nice :)

Reply:
I also thought adding more infos into the popup, for example the file permissions. And also add the possibility to configure the displayed information.